pkgs/top-level/python-aliases.nix

```
@@ -35,16 +35,16 @@ in
 mapAliases ({
   abodepy = jaraco-abode; # added 2023-02-01
   acebinf = throw "acebinf has been removed because it is abandoned and broken."; # Added 2023-05-19
-  adafruit-nrfutil = throw "adafruit-nrfutil has been promoted to a top-level attribute."; # Added 2023-11-19
+  adafruit-nrfutil = throw "adafruit-nrfutil has been promoted to a top-level attribute name pkgs.adafruit-nrfutil."; # Added 2023-11-19
```

```suggestion
  adafruit-nrfutil = throw "adafruit-nrfutil has been promoted to a top-level attribute name:`'pkgs.adafruit-nrfutil`."; # Added 2023-11-19
```

And similar formatting I'd say so that it renders well.